src/test/java/uk/gov/hmcts/reform/sscs/service/CaseFlagMappingServiceTest.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Appeal;
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingOptions;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.YesNo;
+import uk.gov.hmcts.reform.sscs.model.DescendantCaseData;
+
+import java.util.List;
+import java.util.Map;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+public class CaseFlagMappingServiceTest {
+
+    private static final String SIGN_LANGUAGE_TYPE = "Sign Language type";
+
+    @Autowired
+    private CaseFlagMappingService service;
+
+    @Nested
+    class MappingServiceTest {
+
+        @Test
+        void shouldAddTheMappingsGivenTheValuesAreNotNull() {
+            var actual = service.updateHmcCaseData(
+                SscsCaseDataService.builder().sscsCaseData(getSscsCaseData()).build()
+            );
+            var expected = getSscsExpectedMappingCaseData();
+            Assertions.assertEquals(expected, actual);
```

Comment:
this is not an issue in my opinion.

Reply:
I dont think having static imports makes things less verbose, rather it may be detrimental to use static imports as if you have 2 methods with same name from different vendors there will be compiler error.